sys/include/net/lorawan/hdr.h

```diff
@@ -0,0 +1,266 @@
+/*
+ * Copyright (C) 2016 Fundación Inria Chile
```

Comment:
this seems a bit outdated?

Reply:
In theory I produced this file while I was working there. So maybe I should keep it. I will add the HAW Hamburg copyright though

Reply:
resolved? :)